src/figuresClasses.ts

```diff
@@ -1,11 +1,82 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: Function;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape = 'triangle',
```

Comment:
[CODE KNOWLEDGE] When declaring a constructor in TypeScript, you don't need to manually assign parameter values to class properties if you use parameter properties with access modifiers. You can remove the manual assignments and leave the parameters with their access modifiers.